viewer/packages/tools/src/Toolbar/ToolbarTool.test.ts

```diff
@@ -0,0 +1,87 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+import { Cognite3DViewer, CogniteModelBase } from '@reveal/core';
+import { CogniteClient } from '@cognite/sdk';
+import { ToolbarTool } from './ToolbarTool';
+import { createGlContext, mockClientAuthentication } from '../../../../test-utilities';
+import * as THREE from 'three';
+
+describe('ToolbarTool', () => {
+  let viewer: Cognite3DViewer;
+  let models: CogniteModelBase[];
+  let tool: ToolbarTool;
+
+  let canvasContainer: HTMLDivElement;
+
+  const styles = {
+    container: 'reveal-viewer-toolbar-container',
+    bottom: 'reveal-viewer-toolbar-container--bottom'
+  };
+
+  beforeEach(() => {
+    const sdk = new CogniteClient({ appId: 'cognite.reveal.unittest' });
+    mockClientAuthentication(sdk);
+
+    const context = createGlContext(64, 64, { preserveDrawingBuffer: true });
+    const renderer = new THREE.WebGLRenderer({ context });
+    canvasContainer = document.createElement('div');
+    canvasContainer.style.width = '640px';
+    canvasContainer.style.height = '480px';
+
+    viewer = new Cognite3DViewer({ domElement: canvasContainer, sdk, renderer });
+    models = viewer.models.slice();
+    tool = new ToolbarTool(viewer, models[0]);
```

Comment:
How does this work? `viewer.models` will always be empty and `models[0]` would be undefined?

Reply:
See comment below on ToolbarTool and that it shouldnt be bound to a single model